src/fight/mod.rs

```diff
@@ -146,14 +112,313 @@ pub fn resolve_fight(regiment_1: regiment::Regiment, regiment_2: regiment::Regim
     }
 }
 
+/// ## Compute the average damage a unit would dealt to another
+///
+/// ### Paramaters
+/// (&Regiment) first_unit -> The attacker
+///
+/// (&Regiment) second_unit -> The defender
+///
+/// ### Return
+/// (usize, f64) -> A tuple with first the damage computed and then the probability that it occurs
+pub fn compute_mean_case(
+    attacking_regiment: &regiment::Regiment,
+    defending_regiment: &regiment::Regiment,
+) -> (usize, f64) {
+    let damage_probability: f64 = computation_tools::compute_damage_probability(
+        attacking_regiment.get_model(),
+        defending_regiment.get_model(),
```

Comment:
same for the comments about the get_stat
it's basically just replace a call function by an assignment

Reply:
It's cleaner to do so and maybe the function will evolve in the future and call getters more.
And a function call is a jmp operation in assembly and assignement is significantly faster because it's registry stocked